app/assets/javascripts/initializePage.js

```diff
@@ -36,6 +36,27 @@ function callInitializers() {
   initializeDateHelpers();
 }
 
+// This is a helper function that mitigates a race condition happening in JS
+// when this intializer runs before `base.jsx` gets to load the
+// window.Forem / window.Forem.Runtime utility class we depend on for
+// podcast/video playback
+function initializeRuntimeDependantFeatures() {
```

Comment:
Not loving the name of this function but it's what made most sense to me. Suggestions are appreciated if anyone feels it should be named differently 😄